tokens/color/interactive.yaml

```diff
@@ -7,30 +7,121 @@ color:
           Each theme features two shades of blue, one for Link and Focus states and one for Hover
           and Active states.
 
-    blue:
-      lightest:
-        $value: '{color.blue.100}'
-        $description: Inline link hover (dark theme)
-      lighter:
-        $value: '{color.blue.200}'
-        $description: Inline link (dark theme)
-      darker:
-        $value: '{color.blue.400}'
-        $description: Inline link (light theme)
-      darkest:
-        $value: '{color.blue.500}'
-        $description: Inline link hover (light theme)
+    primary:
```

Comment:
Is this syntax correct?

In the docs, is there a way to format this, so that the table for interactive tokens is grouped more like this?
![Screenshot 2023-11-13 at 10 02 28 PM](https://github.com/RedHat-UX/red-hat-design-tokens/assets/95588923/081bfbd2-28f3-4bfc-aa28-311105dcf344)